screens/HomeScreen.js

```diff
@@ -59,6 +59,27 @@ export default class HomeScreen extends React.Component {
       });
   }
 
+  fetchLandmarkDetailsByAddress(address) {
```

Comment:
I'm thinking through this now, and I feel like fetching by address is dangerous. What if there are more than one businesses at that address? It feels like we could get the wrong data back... But the only way around that that I can see is to make sure we always have a google place id in the db, which would require us always looking it up and adding it. what do you think?